reana_cluster/backends/kubernetes/templates/storageclasses/cvmfs.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-cvmfs-cms
+provisioner: csi-cvmfsplugin
+parameters:
+  repository: cms.cern.ch
```

Comment:
Ideally it would be great to have all four LHC experiments supported out of the box, e.g. see on LXPLUS:

```console
$ ls -ld /cvmfs/{alice,atlas,lhcb,cms}.cern.ch
drwxr-xr-x. 14 cvmfs cvmfs 4096 Nov 27  2012 /cvmfs/alice.cern.ch
drwxrwxr-x.  3 cvmfs cvmfs 4096 Mar  5  2012 /cvmfs/atlas.cern.ch
drwxr-xr-x. 58 cvmfs cvmfs 4096 Aug 29  2014 /cvmfs/cms.cern.ch
drwxrwxr-x.  4 cvmfs cvmfs    6 Nov 16  2010 /cvmfs/lhcb.cern.ch
```

(For the future, note also the "resource declaration" idea discussed in https://github.com/reanahub/reana-job-controller/issues/34 and elsewhere where each workflow job would declare which resources it needs, so that they could be either mounted onto pod or the job could be sent to appropriately provisioned nodes... This would however go far beyond the scope of the issue at hand which is to provide a CMS example, or an ATLAS example, or ideally all four LHC experiments, available by default to all.)

Reply:
Added the 4 cvmfs volumes by default, to be changed to on demand mounting specified by each job